src/main/java/com/cupid/jikting/common/aop/LogAspect.java

```diff
@@ -0,0 +1,20 @@
+package com.cupid.jikting.common.aop;
+
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+
+@Slf4j
+@Component
+@Aspect
```

Comment:
p3. 컴포넌트 선언을 클래스에 더 가깝게 하는건 어떨까요?

```suggestion
@Aspect
@Component
```

Reply:
수정완료했습니다